src/lib/components/drs/formComponents/VerifyIdButton.js

```diff
@@ -32,7 +32,7 @@ const VerifyIdButton = props => {
     const handleValid = props.customApiCallSuccessCallback ? props.customApiCallSuccessCallback : defaultHandleValid;
 
     const callAndValidate = relative => {
-        if (relative.id === '') {
+        if (relative.id === '' || relative.id === props.activeDrsObject.id) {
```

Comment:
once you change the prop name in `DrsObjectChildren.js` and `DrsObjectParents.js`, this line will need to be updated with the new prop name.